pkg/config/app/kuma-cp/config.go

```diff
@@ -53,7 +53,6 @@ func (m *Metrics) Validate() error {
 }
 
 type DataplaneMetrics struct {
-	Enabled           bool          `yaml:"enabled" envconfig:"kuma_metrics_dataplane_enabled"`
```

Comment:
Will this field still be accepted (and ignored?)

Reply:
yes